src/App.js

```diff
@@ -67,20 +68,24 @@ const useStyles = makeStyles((theme) => createStyles({
 function App () {
   const classes = useStyles()
   return (
+    <Grid>
+      <Grid className='App-header'>
+        <Grid container className={classes.Container}>
+          <Grid item xs={12} md={12} className={classes.simpleText} />
+          <Grid item xs={12} md={7} className={classes.Table}>
+            {/* timer calling */}
+            <Timer style={{ width: '100%' }} />
 
-    <Grid className='App-header'>
-      <Grid container className={classes.Container}>
-        <Grid item xs={12} md={12} className={classes.simpleText} />
-        <Grid item xs={12} md={7} className={classes.Table}>
-          {/* timer calling */}
-          <Timer style={{ width: '100%' }} />
-
-        </Grid>
-        {/* text field */}
-        <Grid item xs={12} md={5} sm={8} className={classes.text} >
-          <b>Impress the curious <br />Customers with data !</b>
+          </Grid>
+          {/* text field */}
+          <Grid item xs={12} md={5} sm={8} className={classes.text} >
+            <b>Impress the curious <br />Customers with data !</b>
```

Comment:
This is overwriting existing App code, either remove this or talk to Mush and Anagha and figure out a convenient way for them to pick up.
Ideally, this whole thing should be one line like the mainchart